demangle.cc

```diff
@@ -29,6 +32,9 @@ std::optional<std::string_view> cpp_demangle(std::string_view name) {
   static thread_local char *buf;
   static thread_local size_t buflen;
 
+  // TODO(cwasser): Actually demangle Symbols on Windows using e.g.
```

Comment:
My goal is to make mold work exactly the same way on any platform as long as the same command line options and the same set of input files are given. This is important for cross-compilation. To achieve this goal, we eventually need to be able to demangle Itanium-mangled symbols, or otherwise we can't handle `extern "C"` symbols in version scripts.

I think we can just copy demangler source files from libcxxabi.

Reply:
Sounds like a good solution 👍